### PR TITLE
Feature/#5 add pipeline

### PR DESCRIPTION
**Pull request by cristiano-pacheco**
Adds the Github actions to perform the unit tests and code sniffer.

The PHPStan pipeline was skipped because there are a lot of changes to be made and there will be breaking changes. it's worth having a specific issue for it.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
     "name": "run_as_root/ext-magento2-google-shopping-feed",
-    "version": "1.0.0",
```
Comment (riconeitzel) [inline composer.json:3]: we should extract that into a separate PR if this PR takes to long to merge as it fixes a different issue, isn't it?

Comment (cristiano-pacheco) [reply composer.json:3]: Done.

**Review by DavidLambauer — APPROVED** [review]
